tmd/io/tests/test_io.py

```diff
@@ -85,4 +87,11 @@ def test_io_load():
 def test_load_population():
     population = io.load_population(POP_PATH)
     nt.ok_(len(population.neurons) == 5)
-    names = np.array([n.name for n in population.neurons])
+    names = np.array([n.name.split('/')[-1] for n in population.neurons])
+    
+    L = glob.glob(POP_PATH + '/*')
```

Comment:
Test also that files not ending with h5 or swc are being filtered.

Reply:
We don't have any in this directory (since TMD does not support other file formats)

Reply:
That's the point. Add some files with invalid extension (like invalid.extension) and see that it does not crash nor change the number of loaded neurons.